scripts/exrrfs_ioda_bufr.sh

```diff
@@ -17,23 +19,42 @@ yaml_list=(
 "prepbufr_aircar.yaml"
 #"prepbufr_aircft.yaml"
 #"prepbufr_ascatw.yaml"
-#"prepbufr_gpsipw.yaml"
 #"prepbufr_msonet.yaml"
 #"prepbufr_proflr.yaml"
 #"prepbufr_rassda.yaml"
-#"prepbufr_satwnd.yaml"
 #"prepbufr_sfcshp.yaml"
 #"prepbufr_vadwnd.yaml"
 )
 
-# run bufr2ioda.x
+# run bufr2ioda.x for prepbufr data
+# ---------------------------------
 for yaml in ${yaml_list[@]}; do
  sed -e "s/@referenceTime@/${REFERENCE_TIME}/" ${PARMrrfs}/${yaml} > ${yaml}
  source prep_step
  ./bufr2ioda.x ${yaml}
  # some data may not be available at all cycles, so we don't check whether bufr2ioda.x runs successfully
 done
 
+# run python bufr2ioda tool for ZTD and AMV bufr obs
+# ---------------------------------------------
+DIR_ROOT=${HOMErrfs}/sorc/RDASApp/
+${cpreq} ${DIR_ROOT}/rrfs-test/IODA/python/bufr2ioda_ztd.py .
+#${cpreq} ${DIR_ROOT}/rrfs-test/IODA/python/bufr2ioda_satwnd.py .
+${cpreq} ${DIR_ROOT}/rrfs-test/IODA/python/bufr2ioda.json .
+
+USH_IODA=$DIR_ROOT/rrfs-test/IODA/python/
+BUFRJSONGEN=$USH_IODA/gen_bufr2ioda_json.py
```

Comment:
1. Change `BUFRJSONGEN` to lower cases as it is a local variable, not a reserved variable nor an exported variable. And suggest using `py_bufr_json_gen`
2. Use double curly brackets for any SHELL variables:  `${USHioda}`,  not `$USHioda`